src/ripple/app/tx/impl/MPTokenAuthorize.cpp

```diff
@@ -46,68 +46,85 @@ MPTokenAuthorize::preflight(PreflightContext const& ctx)
 TER
 MPTokenAuthorize::preclaim(PreclaimContext const& ctx)
 {
-    auto const sleMptIssuance =
-        ctx.view.read(keylet::mptIssuance(ctx.tx[sfMPTokenIssuanceID]));
-    if (!sleMptIssuance)
-        return tecOBJECT_NOT_FOUND;
-
     auto const accountID = ctx.tx[sfAccount];
     auto const txFlags = ctx.tx.getFlags();
```

Comment:
`txFlags` is used only in one place on line 72. Can just use `ctx.tx.getFlags()` inline instead of declaring a var.